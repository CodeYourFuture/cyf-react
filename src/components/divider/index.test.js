import React from 'react';
import { shallow } from 'enzyme';

import Divider from './';

describe('Divider Component', () => {
  it('renders correctly', () => {
    expect(shallow(<Divider />)).toMatchSnapshot();
  });
});
