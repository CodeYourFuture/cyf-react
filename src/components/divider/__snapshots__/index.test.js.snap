// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider Component renders correctly 1`] = `
<div
  className="divider-1"
>
  <span />
</div>
`;
